.github/workflows/tl500-tech-exercise-test-build-push.yaml

```diff
@@ -0,0 +1,47 @@
+name: tl500-tech-exercise-test-build-push
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - '*'
+    paths:
+      - codereadyworkspaces/tests
+      - .github/workflows/tl500-tech-exercise-test-build-push.yaml
+jobs:
+  build:
+    env:
+      context: test/container
+      image_name: tl500-tech-exercise-test
```

Comment:
@ckavili I think I would need this to be created under quay.io/rht-labs

Reply:
done and done 🚀